incrementalDelivery/v0.1/http.md

```diff
@@ -0,0 +1,7 @@
+# HTTP Request Headers
+
+In order for the server to know that the client can parse the incremental delivery response format, the HTTP request must include the following header:
```

Comment:
@apollographql/clients - I've added the detail we discussed at the team meeting this morning. Any other detail you think is needed here?